_docs/tutorial/00-setup.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Introduction and setup
+overview: This module introduces the tutorial and provides the required setup steps
+
+order: 00
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+# Introduction to microservices with Bookinfo sample application, Kubernetes and Istio
```

Comment:
Don't add any single # titles, the website infra automatically takes the title: and inserts a level 1 heading with it.